src/MSBuild.SDK.SystemWeb.UnitTests/UnitTest1.cs

```diff
@@ -0,0 +1,13 @@
+using Xunit;
```

Comment:
Is this going to have more tests, or is it just a leftover. Think it can be deleted otherwise.

Reply:
You are correct, it's not needed for anything... I'll remove it